### PR TITLE
HOUSNAV-187: initial build wizard work

### PR DESCRIPTION
**Pull request by nicholaslyon-dt**
[HOUSNAV-187](https://hous-bssb.atlassian.net/browse/HOUSNAV-187)

## Overview & Purpose
Create the build wizard page with initial functionality for selecting which walkthroughs to include. As noted in ticket, 'Begin Walkthrough' is hard coded to navigate the current 9.9.9 walkthrough.

## Summary of Changes
- Updated Building Type landing page to match design
- Moved javascript strings for some card content (building type analysis, single dwelling, and multi dwelling) into JSON that is loaded into the application for single source of some content
- Added more structure to json data naming and folders
- Added white colored link variant for the Button component
- Added secondary button variant for the Link component

## Testing
- Navigate to multi dwelling page.
- See default state with 'select all' enabled and 'begin walkthrough' disabled
- Select and deselect walkthroughs and notice updated state of application reflect available functionality

## Checklist
- [x] I have written or updated vitests for this work
- [x] I have reviewed the [accessibility guidelines](https://digital.gov.bc.ca/wcag/home/) relevant to this work
- [x] I have reviewed this work with at least one screen reader (when applicable)
- [x] I have added/updated any related documentation

### CODE REVIEW

**Review by stechkev — APPROVED** [review]

File: packages/data/json/building-types/single-dwelling/wt-single-dwelling-9.9.9.test.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
 // 3rd party
 import { describe, it, expect } from "vitest";
 // local
-import { isWalkthroughItemTypeVariable } from "../../src/useWalkthroughData";
-import data from "./single-dwelling-9.9.9.json";
+import { isWalkthroughItemTypeVariable } from "../../../src/useWalkthroughData";
+import data from "./wt-single-dwelling-9.9.9.json";
```
Comment (jbook24) [inline packages/data/json/building-types/single-dwelling/wt-single-dwelling-9.9.9.test.ts:5]: what is the wt- stand for?

File: packages/data/json/building-types/wt-building-type-analysis.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
+{
+  "info": {
+    "title": "Building Type Analysis",
+    "subtitle": "",
+    "description": "Not sure? Narrow down whether your project falls under the specifications of a single- or a multi-unit dwelling.",
+    "startingSectionId": "1_intro",
+    "walkthroughTitle": "FPO - Building Type Analysis"
```
Comment (jbook24) [inline packages/data/json/building-types/wt-building-type-analysis.json:7]: What is FPO stand for?

File: packages/constants/src/urls.ts [hunk 3]
```diff
@@ -50,48 +47,33 @@ export const URLS_FOOTER = [
...
-    };
-    return arr;
-  },
-  {} as Record<EnumBuildingTypes, UrlType>,
-);
-
 export const URLS_WALKTHROUGHS: Record<
   EnumBuildingTypes,
-  Record<EnumWalkthroughIds, UrlType>
+  Record<EnumWalkthroughIds, string>
```
Comment (jbook24) [inline packages/constants/src/urls.ts:61]: Could this change impact type safety?

**Review by jbook24 — APPROVED** [review]

File: packages/data/json/building-types/single-dwelling/wt-single-dwelling-9.9.9.test.ts (hunk 1 above)
Comment (nicholaslyon-dt) [reply packages/data/json/building-types/single-dwelling/wt-single-dwelling-9.9.9.test.ts:5]: shorthand for walkthrough. I wanted to differentiate between JSON that represented a specific walkthrough vs other site JSON.

File: packages/data/json/building-types/wt-building-type-analysis.json (hunk 2 above)
Comment (nicholaslyon-dt) [reply packages/data/json/building-types/wt-building-type-analysis.json:7]: I always thought it was "for proof only" but according to Wikipedia that's not it 😂. 

https://en.wikipedia.org/wiki/For_position_only

Either way it's just a note that the text here isn't final.

File: packages/constants/src/urls.ts (hunk 3 above)
Comment (nicholaslyon-dt) [reply packages/constants/src/urls.ts:61]: Not really as I removed the need for that part to be an object and instead is now just a string. Either way, this whole variable is going away anyway as I build out the idea of multi walkthroughs. I've already removed it locally as I've been working on HOUSNAV-200

**Pull request merged**